spec/abstract/ARM/ArchTcb_A.thy

```diff
@@ -37,8 +37,8 @@ where
 definition arch_post_set_flags :: "obj_ref \<Rightarrow> tcb_flags \<Rightarrow> (unit, 'a::state_ext) s_monad" where
   "arch_post_set_flags t flags \<equiv> return ()"
 
-(* This is defined here in comparision to its location in HYP architectures due to not causing a
-   circular dependency by calling functions that have to be defined later. *)
+(* As opposed to HYP architectures, we can keep this here as it does not cause circular function
+   dependency. *)
```

Comment:
I'd say "On non-hyp architectures, we ..."

Reply:
(same for other arches)

Reply:
Sure, I'm happy with that